### ISSUE TITLE
[proposal] plugins in `resmgr` module should be moved to `runtimehooks` or `qos-manager`

### ISSUE THREAD

**Issue by zwzhang0107** [koordinator-sh/koordinator#174]
**What is your proposal**:
Plugins in `resmgr` can be classified into two types: `static` and `dynamic`.
For `static` type, advanced parameters like NodeSLO will be injected according to pod QoS and Priority, such as cpu qos, memory qos, 
For `dynamic` type, pod will be adjusted dynamically according to node running state, such as cpu suppress, cpu burst, memory evict.

**Why is this needed**:
move `static` type to `runtime hook`
- [x] cpu qos(group identity)
- [ ] [memory qos](https://github.com/koordinator-sh/koordinator/issues/755)
- [ ] [LLC/MBA isolation](https://github.com/koordinator-sh/koordinator/issues/756)
- [x] [extend-resource settings(request and limit of batch-cpu/memory)](https://github.com/koordinator-sh/koordinator/issues/717)

move `dynamic` type to `qos-manager`, which is an enhancement of `resmgr` with well-designed framework.
- [x] [cpu burst](https://github.com/koordinator-sh/koordinator/issues/757)
- [x] [cpu suppress](https://github.com/koordinator-sh/koordinator/issues/758)
- [x] [evict strategy by memory](https://github.com/koordinator-sh/koordinator/issues/759)
- [x] [evict strategy by cpu](https://github.com/koordinator-sh/koordinator/issues/760)

**Is there a suggested solution, if so, please add it**:

**Comment by jasonliu747** [comment]
blocked by #58

**Comment by ZiMengSheng** [comment]
The reorganization this proposal asked for has landed: `pkg/koordlet/resmgr` no longer exists — its responsibilities were split into `qosmanager` (dynamic strategies: cpu suppress/burst, memory evict, ...) and `runtimehooks` (static injection: cpu qos/group identity, batchresource, cpuset, ...). Closing as completed. If you see remaining gaps in the new structure, please open a new issue.
